client/src/components/buttons/GameModeButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import classes from "../../styles/DropDownPanel.module.css"
+export default class GameModeButton extends React.Component  {
+    handleClick = () => {
+        console.log(this.props.text);
+    }
+    render() {
+        return (
+            <button onClick={ this.handleClick } 
+                    className={classes.modeButton}>
+                {this.props.text}
+            </button>
+        );
+    }
+}
```

Comment:
Лучше использовать либо только ClassBased компоненты, либо только FuncitonBased
```suggestion

export default function GameModeButton({text}) {
    function handleClick() {
        console.log(text);
    }
    return (
        <button onClick={ handleClick } 
                className={classes.modeButton}>
            {text}
        </button>
    );
}
```